sys/mips/ebase.S

```diff
@@ -396,8 +397,11 @@ user_exc_leave:
         sw      t0, PCPU_SR(s0)
 
         # If user has FPU disabled, don't restore its registers.
```

Comment:
This comment needs updating: now we also skip restoring the FPU registers if we know the current FPU context belongs to the current thread (i.e. there were no context switches).

Reply:
done